cms/src/components/content/ContentOverview.spec.ts

```diff
@@ -102,6 +102,54 @@ describe("ContentOverview.vue", () => {
         });
     });
 
+    it("should display search input", async () => {
+        const wrapper = mount(ContentOverview, {
+            global: {
+                plugins: [createTestingPinia()],
+            },
+            props: {
+                docType: DocType.Post,
+            },
+        });
+
+        await waitForExpect(() => {
+            const searchInput = wrapper.find('[data-test="search-input"]');
+            expect(searchInput.exists()).toBe(true);
+        });
+    });
+
+    it("should display sort options when sort-button is clicked", async () => {
+        const wrapper = mount(ContentOverview, {
+            global: {
+                plugins: [createTestingPinia()],
+            },
+            props: {
+                docType: DocType.Post,
+            },
+        });
+
+        //data-test="sort-option-title"
+
+        await waitForExpect(async () => {
+            const sortToggleBtn = wrapper.find('[data-test="sort-toggle-btn"]');
+            expect(sortToggleBtn.exists()).toBe(true);
+            await sortToggleBtn.trigger("click");
+
+            const sortOptionsDiv = wrapper.find('[data-test="sort-options-display"]');
+            expect(sortOptionsDiv.exists()).toBe(true);
```

Comment:
I think this is of limited utility: testing that the actual options are displayed is fine, although it would perhaps be more valuable to test their functionality. I don't think it really matters that their container div is also rendered - perhaps we refactor the HTML in the future, and then this test would fail, even though the functionality could still work.

Reply:
Functionality testing was added